src/_premake_init.lua

```diff
@@ -1650,7 +1651,7 @@
 
 	-- Windows and friends.
 
-	filter { "system:Windows or language:C#", "kind:ConsoleApp or WindowedApp" }
+	filter { "system:Windows or language:C# or language:F#", "kind:ConsoleApp or WindowedApp" }
```

Comment:
The language part of this filter just doesn't seem right? If I build a C#/F# project on Linux, I expect the output file to not be `.exe` unless it's cross-compiling for Windows. Can anyone elaborate as to why this filter contains languages?

Reply:
.NET executables end up in `.exe`, even in Linux.

Reply:
Agree, @samsinsane you end up feeding mono with the exe to start the application.

Reply:
@tritao @realvictorprm Thanks for the clarification!